citations-api/api/src/java/org/sakaiproject/citation/api/Validator.java

```diff
@@ -0,0 +1,11 @@
+package org.sakaiproject.citation.api;
+
+import java.util.List;
+
+/**
+ * Created by nickwilson on 9/29/15.
+ */
+public interface Validator {
```

Comment:
Having it called `CitationValidator` means you don't need to ugly fully qualified use in the tool.